inputs/prod_inner.json

```diff
@@ -1,4 +1,5 @@
 {
+  "version": "{{VERSION}}",
```

Comment:
Is `version` descriptive enough or should it be `client-version`? To me `version` sounds like it might be specified a schema version or something.